### PR TITLE
Add Row and Column components

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

**Review by aabounegm — COMMENTED** [review]
I think you may have misunderstood what I meant by row and column. What you implemented is similar to the way Flutter does the layout, but what I had in mind was [Vuetify](https://vuetifyjs.com/en/components/grids/#sub-components)'s `v-row` and `v-col`, which is more in line with the convention used more in the web where the row is divided into 12 columns, as is also the case with [Bootstrap](https://getbootstrap.com/docs/5.0/layout/grid/), [Skeleton](http://getskeleton.com/#grid:~:text=THE%20GRID), [Tailwind](https://tailwindcss.com/docs/grid-template-columns), and many others, typically with a breakpoint system as well for responsiveness (but that can come later).

File: docs/src/routes/docs/_layout.svelte [hunk 1]
```diff
@@ -75,6 +75,10 @@
           title: 'Chip',
           segment: 'components/chip',
         },
+        {
+          title: 'Column',
+          segment: 'components/column',
+        },
```
[the remark this answers is not in the document]
Comment (illright) [reply docs/src/routes/docs/_layout.svelte:81]: Definitely not now, it would be rather ugly. Maybe after the redesign lol

**Comment by illright** [comment]
In that case, what would be the components?

**Comment by aabounegm** [comment]
Also `Row` and `Column`, but their usages and props would be different. `Column` can only be used directly inside `Row`, and would have props determining how many columns (out of 12) to take up. Take a look at [`v-row`](https://vuetifyjs.com/en/api/v-row/) and [`v-col`](https://vuetifyjs.com/en/api/v-col/) APIs for ideas.

**Pull request closed without merging**